internal/executor/job/submit_test.go

```diff
@@ -94,3 +94,30 @@ func newArbitraryError(message string) error {
 func newArmadaErrCreateResource() error {
 	return &armadaerrors.ErrCreateResource{}
 }
+
+func TestSubmitJobs(t *testing.T){
+	submitService := &SubmitService{
```

Comment:
I think running `go fmt` on this file would collapse the two lines of the SubmitService reference into one, since there are no members declared in it.